src/_includes/svg/github.svg

```diff
@@ -1,5 +1,6 @@
 <svg role="img" viewBox="0 0 24 24"
```

Comment:
For all of the svgs:

- [ ] Remove `class={{svgStyleClass}}` for a future time
- [ ] Ensure `role="img"` is set (it is here, just make sure to check them all)
- [ ] Add a unique `id` to the `<title>` element. Eg `<title id="githubSvgTitle">...</title>`
- [ ] Add `aria-labelledby="uniqueTitleID"` to the `<svg>` element
- [ ] Define a default title - this can be done inline using [nunjucks expressions](https://mozilla.github.io/nunjucks/templating.html#if-expression). For this one specifically you could do this like this:
  ```njk
  <title id="githubSvgTitle">{{svgTitle if svgTitle is defined else "GitHub"}}</title>
  ```

So this one (`github.svg`) would become:

```
<svg
  role="img"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
  aria-labelledby="githubSvgTitle"
>
  <title id="githubSvgTitle">{{svgTitle if svgTitle is defined else "GitHub"}}</title>
  ...
</svg>
```

Check out [this article on CSS Tricks](https://css-tricks.com/accessible-svgs/) for some more info behind the a11y changes!